### PR TITLE
typo changed

### PR DESCRIPTION
**Pull request by abhishekjha812**
Fixes #

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by vsasvipul0605 — APPROVED** [review]

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
__not showing date and time.__

add to calender

File: app/components/public/add-to-calender.hbs [hunk 1]
```diff
@@ -5,7 +5,7 @@
       {{#if this.isSingleDay}}
         {{general-date @event.startsAt "dddd, D MMMM, YYYY" tz=@event.timezone}}<br>{{general-date @event.startsAt "h:mm A" tz=@event.timezone}} {{t 'to'}} {{general-date @event.endsAt "h:mm A" tz=@event.timezone}} 
       {{else}}
-        {{general-date @event.startsAt "dddd, D MMMM, YYYY h:mm A" tz=@event.timezone}} {{t 'to'}} {{general-date @event.endsAt "dddd, D MMMM, YYYY h:mm A" tz=@event.timezone}} 
+        {{general-date @event.startsAt 'date-time-long' tz=@event.timezone}} {{t 'to'}} {{general-date @event.endsAt 'date-time-long' tz=@event.timezone}}
```
Comment (maze-runnar) [inline app/components/public/add-to-calender.hbs:8]: that is not  a typo, it's format we have switched In a PR.

**Comment by daretobedifferent18** [comment]
@iamareebjamal Pls close this PR. Not needed

**Comment by iamareebjamal** [comment]
How come random people approve the most ridiculuous PRs and never review legitimate ones

**Comment by abhishekjha812** [comment]
> How come random people approve the most ridiculuous PRs and never review legitimate ones

sir! I am new to open source and i was just exploring , I had no intensions to disobey the open source ethics 
if you have any suggestions for me then please share it :)

**Comment by daretobedifferent18** [comment]
@abhishekjha812 Pls if you want to work on any issue, work on open issues and if you think there is an issue besides opened issues then pls first create a new issue.

**Pull request closed without merging**